arches_lingo/src/arches_lingo/components/scheme/report/SchemeNamespace.vue

```diff
@@ -35,13 +45,35 @@ onMounted(async () => {
 });
 
 async function save() {
-    await updateSchemeNamespace(
-        route.params.id as string,
-        schemeInstance.value as SchemeInstance,
-    );
+    try {
+        let updated;
+        if (route.params.id === NEW) {
+            updated = await createScheme(schemeInstance.value);
+            router.push({
+                name: "scheme",
+                params: { id: updated.resourceinstanceid },
+            });
+        } else {
+            updated = await updateSchemeNamespace(
+                route.params.id as string,
+                schemeInstance.value,
+            );
+        }
+        schemeInstance.value = updated;
+        emit(UPDATED);
```

Comment:
This appears to work for now, but the update gets emitted before the router pushes the minted pk into the route, which is what the sections are using for the source of truth of the resource instance id. We might want to look into some alternatives -- e.g., if we ever implement backend logic to create additional tiles when a different tile is posted, then we could run into trouble with staleness of data in other sections, because they're not fetching data ("/new" is still in the route).